app/package.json

```diff
@@ -90,7 +90,8 @@
     "relay-runtime": "^13.2.0",
     "styled-components": "^5.3.5",
     "typography": "^0.16.21",
-    "validator": "^13.9.0"
+    "validator": "^13.9.0",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.19.3/xlsx-0.19.3.tgz"    
```

Comment:
Probably better to just use the npm registry rather than the CDN here, atl east for consistency's sake

Reply:
I had to use cdn - see https://docs.sheetjs.com/docs/getting-started/installation/nodejs/

> Older releases are technically available on the public npm registry as xlsx, but the registry is out of date. The latest version on that registry is 0.18.5
> This is a known registry bug

Reply:
Ah sorry didn't know this, thanks for letting me know!